int8 status
string parameter_info